mne/io/nirx/nirx.py

```diff
@@ -288,19 +288,20 @@ def prepend(list, str):
             raw_extras=[raw_extras], verbose=verbose)
 
         # Read triggers from event file
-        with _open(files['evt']) as fid:
-            t = [re.findall(r'(\d+)', line) for line in fid]
-        onset = np.zeros(len(t), float)
-        duration = np.zeros(len(t), float)
-        description = [''] * len(t)
-        for t_idx in range(len(t)):
-            binary_value = ''.join(t[t_idx][1:])[::-1]
-            trigger_frame = float(t[t_idx][0])
-            onset[t_idx] = (trigger_frame) * (1.0 / samplingrate)
-            duration[t_idx] = 1.0  # No duration info stored in files
-            description[t_idx] = int(binary_value, 2) * 1.
-        annot = Annotations(onset, duration, description)
-        self.set_annotations(annot)
+        if op.isfile(files['hdr'][:-4]+'.evt'):
+            with _open(files['hdr'][:-4]+'.evt') as fid:
```

Comment:
Surround `+` with spaces. Also, you can re-use the dot from `files['hdr'][:-3]`. Bonus comment: using `pathlib` would expose the file stem and extension without having to do any string slicing (`pathlib` would actually be a better replacement for `os.path` - but this could be done in a separate PR).

Reply:
Thanks